packages/components/src/panel/test/__snapshots__/row.js.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PanelRow should render with the custom class name 1`] = `undefined`;
```

Comment:
Is this correct?

Reply:
Of course not 🙈 

Thanks for spotting it! We were missing a `render()` in the test, so this test was using the snapshot from the previous render.

Fixed in c743bdf142f9d2b5dc585bec4e582fb560f58779